package.json

```diff
@@ -69,7 +69,7 @@
     "size-limit": "^4.9.2",
     "stylelint": "^13.11.0",
     "stylelint-config-standard": "^20.0.0",
-    "typescript": "^4.1.5"
+    "typescript": "^4.3.5"
```

Comment:
Kind of annoying, although not that big a deal, to have to specify typescript in 2 places.

I think it's because the top level needs it for linting, and packages/components needs it for type checking (which I added, so it's my own damn fault).

We could maybe set things up so it's only needed at the top level. This is also the kind of yarn workspaces handles nicely, since deps are only installed at the top level.

But not worth worrying about too much right now.

Reply:
👍  tracking: https://app.clubhouse.io/czi-edu/story/151292/only-import-typescript-at-the-top-level